src/App.js

```diff
@@ -1,25 +1,53 @@
-import logo from './logo.svg';
 import './App.css';
+import firebase from './firebase';
```

Comment:
I really like your app, Joey! The use of api and firebase is great! Sorry I don't have any comments :)